suse_migration_services/defaults.py

```diff
@@ -39,7 +39,7 @@ def get_migration_log_file(self):
         )
 
     @classmethod
-    def get_system_migration_debug_file(self):
+    def get_system_migration_opt_config_file(self):
```

Comment:
Can we name this differently ? e.g

get_system_migration_custom_config_file()

Reply:
Fixed.